resources/eventing/charts/nats/README.md

```diff
@@ -32,49 +40,60 @@ nats:
 
   # How many seconds should pass before sending a PING
   # to a client that has no activity.
-  pingInterval: 
+  pingInterval:
 
   # Server settings.
   limits:
-    maxConnections: 
-    maxSubscriptions: 
-    maxControlLine: 
-    maxPayload: 
+    maxConnections:
+    maxSubscriptions:
+    maxControlLine:
+    maxPayload:
 
-    writeDeadline: 
-    maxPending: 
-    maxPings: 
-    lameDuckDuration: 
+    writeDeadline:
+    maxPending:
+    maxPings:
+    lameDuckDuration:
 
   # Number of seconds to wait for client connections to end after the pod termination is requested
   terminationGracePeriodSeconds: 60
 ```
 
 ### Logging
 
-> **NOTE**: It is not recommended to enable trace or debug in production, since enabling it will significantly degrade performance.
+*Note*: It is not recommended to enable trace or debug in production since enabling it will significantly degrade performance.
 
 ```yaml
 nats:
   logging:
-    debug: 
-    trace: 
-    logtime: 
-    connectErrorReports: 
-    reconnectErrorReports: 
+    debug:
+    trace:
+    logtime:
+    connectErrorReports:
+    reconnectErrorReports:
 ```
-### Clustering
 
-If clustering is enabled, then a 3-node cluster will be set up. You can find more information in the [NATS documentation](https://docs.nats.io/running-a-nats-service/introduction/running/nats-kubernetes/helm-charts#clustering).
+## Clustering
+
+If clustering is enabled, then a 3-node cluster will be setup. More info at:
+https://docs.nats.io/nats-server/configuration/clustering#nats-server-clustering
 
 ```yaml
 cluster:
   enabled: true
   replicas: 3
 ```
-### JetStream
 
-To set up memory and file storage with JetStream, use this configuration: 
+Example:
+
+```sh
+$ helm install nats nats/nats --set cluster.enabled=true
```

Comment:
have you tested this command with our helm chart? if its not working, then remove it from here.

Reply:
removed it.